renku/service/serializers/cache.py

```diff
@@ -210,20 +207,30 @@ class ProjectListResponseRPC(JsonRPCResponse):
     result = fields.Nested(ProjectListResponse)
 
 
-class ProjectMigrateRequest(RepositoryContext):
+class ProjectMigrateRequest(AsyncSchema, CommitSchema, LocalRepositorySchema, RemoteRepositorySchema):
     """Request schema for project migrate."""
 
     force_template_update = fields.Boolean(default=False)
     skip_template_update = fields.Boolean(default=False)
     skip_docker_update = fields.Boolean(default=False)
     skip_migrations = fields.Boolean(default=False)
 
+    # ! temporary -- should be removed but there are tests to update
```

Comment:
Did you update the tests?

Reply:
There is this one I'm not sure how to fix https://github.com/SwissDataScienceCenter/renku-python/blob/master/tests/service/jobs/test_project.py#L26-L51
Removing the `migrate_project` makes it fail in the `delayed_ctrl_job` function. I can't understand the goal of the test (the description is probably a wrong copy/paste from another test).
Is it testing the behavior of `"migrate_project": True` and should I use another serializer then `ProjectMigrateRequest` ?